open_prices/prices/models.py

```diff
@@ -463,12 +463,6 @@ def clean(self, *args, **kwargs):
                 )
 
             if proof:
-                if proof.owner != self.owner:
```

Comment:
we should not allow users to add prices on proofs they do not own such as RECEIPTs or GDPR_REQUESTs or SHOP_IMPORTs

Reply:
True!